dtreeviz/trees.py

```diff
@@ -915,14 +923,21 @@ def rtree_feature_space3D(self,
                               fontsize=10, ticks_fontsize=8, fontname="Arial",
                               azim=0, elev=0, dist=7,
                               show={'title'}, colors=None, markersize=15,
-                              n_colors_in_map=100, figsize=None, ax=None):
+                              n_colors_in_map=100,
+                              features=None,
+                              figsize=None, ax=None):
         """
         Show 3D feature space for bivariate regression tree. X_train should have
         just the 2 variables used for training.
         """
-        _rtreeviz_bivar_3D(self.shadow_tree, fontsize, ticks_fontsize, fontname, azim, elev, dist, show, colors,
-                           markersize,
-                           n_colors_in_map, figsize, ax)
+        if features is None:
+            features = self.shadow_tree.feature_names[0:2] # pick first two features if none given
```

Comment:
@parrt do you want to add a safety check here as well, or just let it crash on the user if they have < 3 features?

Reply:
oh crap, i missed one? hang on.  done.